shared/wallets/routes.js

```diff
@@ -199,5 +197,7 @@ export const newRoutes = {
 }
 
 export const newModalRoutes = {
+  reallyRemoveAccount: {getScreen: () => require('./wallet/settings/popups').ReallyRemoveAccountPopup},
```

Comment:
Did this so that the modals are centered with respect to the whole window, and not just the wallet screen.

Reply:
Actually, I'll just do this in a separate PR where I move all the modals.